apps/challenges/migrations/0038_challengephase_max_submissions_per_month.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
@RishabhJain2018 
This is because there was Migration Conflict when I had [Merge branch 'master' into issue#1690](https://github.com/Cloud-CV/EvalAI/pull/1886/commits/872a34fb02636d104805a2fa4466269ba8a81c0d) and Commit to [Resolve Migration Conflicts](https://github.com/Cloud-CV/EvalAI/pull/1886/commits/ecc8213817be0a10f95042a41e5d56446b077a7d) because of addition of `0038_challenge_remote_evaluation.py`.

Reply:
Resolved